bootstrap.cmd

```diff
@@ -3,6 +3,7 @@
 where /q dotnet.exe || echo Can't find dotnet.exe! Please add to PATH && goto :EOF
 where /q git.exe || echo Can't find git.exe! Please add to PATH && goto :EOF
 set root=%~dp0
+pushd %root%
```

Comment:
Seems fine.  The semantics are "put the jitutils dir where the script is" right?  Rather than where it's invoked from.